packages/apps/reputation-oracle/server/src/modules/webhook/webhook-incoming.module.ts

```diff
@@ -5,26 +5,27 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { HttpModule } from '@nestjs/axios';
 import { Web3Module } from '../web3/web3.module';
 import { WebhookController } from './webhook.controller';
-import { WebhookService } from './webhook.service';
 import { WebhookIncomingEntity } from './webhook-incoming.entity';
-import { WebhookOutgoingEntity } from './webhook-outgoing.entity';
 import { WebhookIncomingRepository } from './webhook-incoming.repository';
-import { WebhookOutgoingRepository } from './webhook-outgoing.repository';
+import { EscrowCompletionTrackingRepository } from '../escrow-completion-tracking/escrow-completion-tracking.repository';
+import { EscrowCompletionTrackingModule } from '../escrow-completion-tracking/escrow-completion-tracking.module';
+import { WebhookIncomingService } from './webhook-incoming.service';
 
 @Module({
   imports: [
-    TypeOrmModule.forFeature([WebhookIncomingEntity, WebhookOutgoingEntity]),
+    TypeOrmModule.forFeature([WebhookIncomingEntity]),
     ConfigModule,
     Web3Module,
     HttpModule,
+    EscrowCompletionTrackingModule,
   ],
   controllers: [WebhookController],
   providers: [
     Logger,
-    WebhookService,
+    WebhookIncomingService,
     WebhookIncomingRepository,
-    WebhookOutgoingRepository,
+    EscrowCompletionTrackingRepository,
```

Comment:
Should it be service and not repository?

I might need to check deeper how nest.js modules system works, but seems that we don't need that at all because it imports `EscrowCompletionTrackingModule` above

Reply:
Yes, you are right, we don't need to provide `EscrowCompletionTrackingModule` here.